ansys/mapdl/reader/cython/_binary_reader.pyx

```diff
@@ -364,7 +364,7 @@ cdef np.ndarray wrap_array(void* c_ptr, int size, int type_flag, int prec_flag):
     array_wrapper.set_data(size, c_ptr, my_dtype)
 
     cdef np.ndarray ndarray
-    ndarray = np.array(array_wrapper, copy=False)
+    ndarray = np.array(array_wrapper)
 
```

Comment:
Couldn't we resolve this based on the numpy version used? Preventing users from using numpy 1.x only for this change seems a bit excessive IMO. Many libraries are still struggling with the switch to numpy 2.X so this might be an issue.